frontend/package.json

```diff
@@ -25,6 +25,7 @@
     "test": "jest"
   },
   "devDependencies": {
+    "@apollo/client": "^3.2.7",
```

Comment:
I think that pretty much everything in this file is wrong. You are adding more of the same "wrongdoing" to this file which may work but we will have to fix that.

Neither or dev dependencies nor our peer dependencies make sense.

Reply:
Agreed, but if you're OK we'll handle this issue in a separate PR. It's not worse than what was done before, so I don't think this should block merging this as long as we plan to fix the global issue soon.